src/main/kotlin/com/cosmotech/api/config/CsmPlatformProperties.kt

```diff
@@ -83,10 +83,26 @@ data class CsmPlatformProperties(
     /** Queue configuration */
     val eventbus: CsmEventBus
 ) {
-  data class CsmStorage(val host: String, val reader: CsmStorageUser, val admin: CsmStorageUser) {
+  data class CsmStorage(
+      val host: String,
+      val port: String = "5432",
+      val reader: CsmStorageUser,
+      val writer: CsmStorageUser,
+      val admin: CsmStorageUser
+  ) {
     data class CsmStorageUser(val username: String, val password: String)
   }
-  data class CsmEventBus(val host: String, val username: String, val password: String)
+  data class CsmEventBus(
+      val host: String,
+      val port: String = "5672",
```

Comment:
So we are going with host+port instead of the baseUri? As long as we can pass the correct URL with the proper scheme to the simulator I'm fine.